hri_safety_sense/src/VscProcess.cpp

```diff
@@ -15,9 +15,8 @@
 /**
  * ROS Includes
  */
-#include "ros/ros.h"
-#include "sensor_msgs/Joy.h"
-#include "std_msgs/UInt32.h"
+#include "rclcpp/rclcpp.hpp"
+#include "sensor_msgs/msg/joy.hpp"
 
 /**
  * System Includes
```

Comment:
Make sure to include what you are using here; this includes `<chrono>`, `<functional>`, `<memory>`, as well as `<std_msgs/msg/UInt32.hpp>`.

Reply:
7de7162